generators/app/templates/README.ejs

```diff
@@ -94,56 +93,17 @@ Make sure you have the latest version of the Xcode command line tools installed:
 ```
 xcode-select --install
 ```
-Follow this guides to properly install rbenv and ruby-build. These are tools that will easily allow you to install and change different ruby versions.
-1- [Rbenv](https://github.com/rbenv/rbenv#installation)
-2- [Ruby build](https://github.com/rbenv/ruby-build#installing-as-an-rbenv-plugin-recommended)
+Follow this guides to properly install rvm. These are tools that will easily allow you to install and change different ruby versions.
```

Comment:
`Follow this [guide](https://github.com/rvm/rvm) to properly install rvm. This tool will easily allow you to install and change different ruby versions.`